src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1886,33 +1886,38 @@ MergeOperation MergeWhenNotMatched() : {
         { return mi; }
 }
 
-List<String> RelObjectNameList() : {
+ObjectNames RelObjectNames() : {
     String token = null;
+    Token delimiter = null;
     List<String> data = new ArrayList<String>();
+    List<String> delimiters = new ArrayList<String>();
 } {
     token = RelObjectNameExt() { data.add(token); }
-    ( LOOKAHEAD (2) ("." | ":") ("." { data.add(null); })* token = RelObjectNameExt2() { data.add(token); } ) *
+    (
+        LOOKAHEAD (2) ( delimiter = "." | delimiter = ":" ) { delimiters.add(delimiter.image); } ( delimiter =  "." { data.add(null); delimiters.add(delimiter.image); })*
```

Comment:
I _think_ we should add the colon to the optional part here as well, but it wasn't there before I made the change (`("." { data.add(null); })*`), so just wanted to double check that's OK? If it's OK, the line will read 
```
LOOKAHEAD (2) ( delimiter = "." | delimiter = ":" ) { delimiters.add(delimiter.image); } ((delimiter =  "." | delimiter = ":") { data.add(null); delimiters.add(delimiter.image); })*

Reply:
Looks consistent to me, I would do the same.

Reply:
Done